### DISCUSSION TITLE
Does your woff and woff2 comply with OFL?

### DISCUSSION THREAD

**Discussion by krystofmatejka** [fontsource/fontsource#594] [Q&A] [answered]
Hello, if you convert ttf under the OFL to a different format, you have to meet certain criteria to not break the license. https://opensource.stackexchange.com/questions/9673/does-converting-a-google-font-to-another-format-breach-the-license

Long story short, you must not modify the font with the exception of compression. That also means you do not have to remove any metadata or add any additional metadata.

From your documentation is not clear whether your woff and woff2 files meet these criteria. Can you please clarify it?

**Comment by ayuhito** [comment, marked as answer]
We don't modify anything as we practically mirror the Google Fonts repository. If they're not breaking any rules, we aren't either.

For non-Google fonts, we only apply compression for some rare cases where their WOFF variants aren't publicly available. I don't believe we have any non-Google fonts that also have RFNs.

**Reply by krystofmatejka** [reply]
Thank you for your answer. Can you specify what you mean by mirroring? You copy the ttf font and then convert it to woff2 by some tool right?

My main concern is if the OFL license is not violated during the conversion.

**Reply by ayuhito** [reply]
Ah, we don't perform any conversions there either. We fetch from the Google Fonts CSS API directly, so Google has already done the WOFF conversions for us anyways.

The underlying library responsible for this is [google-font-metadata](https://github.com/fontsource/google-font-metadata) if you want more understanding of this.

**Reply by krystofmatejka** [reply]
Thank you very much for the clarification.
